spec/takeaway_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  context 'when initialized' do
+    it 'accepts 1 argument: default menu' do
+      menu = double()
+      expect(subject).to respond_to(:menu)
+    end
+    describe '#show_menu' do
```

Comment:
As a next step, you can write another test for show_menu, to not only test that the method exists, but on what it does.